### PR TITLE
Updated to simplify readme to the basic information such as links to …

### LINKED ISSUE #64: Update completely outdated README.rst
Should provide a brief overview, then point to user stories, design documents, and other docs. Currently the README reflects team's decisions from a couple of months back, and it is very much out of date.

### PR DESCRIPTION
**Pull request by ChristopherMSpencer**
+Craton brief description
+Project Resources: License, docs, source, bugs, wiki links. 
+Links to project status and backlog
-Excess information that will be moved to a wiki.

Resolves #64

### CODE REVIEW

**Comment by sigmavirus24** [comment]
This looks good to me.

File: README.rst [hunk 1]
```diff
@@ -1,157 +1,52 @@
...
-craton
-===============================
+===================================
+Craton (OpenStack Fleet Management)
+===================================
 
-Fleet management for OpenStack.
+Craton is a fleet management tool set that helps operators manage
+their OpenStack cloud environments.  Craton provides operators with
+monitoring and auditing to provide reports and alerts on resource status
```
Comment (sulochan) [inline README.rst]: we don't provide monitoring.

File: README.rst [hunk 2]
```diff
@@ -1,157 +1,52 @@
...
+to help maintain cloud health. Craton uses a relational database to persist
+inventory information. SQLAlchemy will be used for modeling, with Alembic
+supporting migrations.  The inventory database is completely encapsulated
+by a CLI, REST, and python API.
```
Comment (sulochan) [inline README.rst]: Same with this ... i know we might have said something like this in the previous version but i would rather we not put any of this here. Some of this info does not need to be mentioned here at all .. like sqlalchemy, alembic etc.

File: README.rst [hunk 3]
```diff
@@ -1,157 +1,52 @@
...
 
-Python API
-==========
+.. TODO(cmspence) Wiki: https://wiki.openstack.org/wiki/Craton
 
-Wraps the REST API above.
+Project status, bugs, and features are tracked in GitHub as well as Waffle.io:
 
+.. TODO(cmspence) Project status, bugs, and features are tracked on Launchpad:
+ http://launchpad.net/craton
```
Comment (sulochan) [inline README.rst]: No need to make this a todo .. i suggest we just put launchpad now .. and also update the rest of this doc to reflect upstream stuff .. like wiki in openstack already exists .. bugs etc on launchpad etc.

File: README.rst (hunk 1 above)
Comment (ChristopherMSpencer) [inline README.rst]: fixed

File: README.rst (hunk 2 above)
Comment (ChristopherMSpencer) [inline README.rst]: fixed

File: README.rst (hunk 3 above)
Comment (ChristopherMSpencer) [inline README.rst]: fixed

**Comment by sulochan** [comment]
lgtm

**Pull request merged**